tmt/steps/provision/mrack.py

```diff
@@ -229,6 +229,18 @@ def _transform_beaker_pool(
         actual_value)
 
 
+def _transform_cpu_family(
+        constraint: tmt.hardware.NumberConstraint,
```

Comment:
Should be `IntegerConstraint`, CPU family will never be a floating point number.